annotations.Rmd

```diff
@@ -92,10 +112,11 @@ You can map data values to these aesthetics, but use restraint: it is hard to pe
     (Note that I manually tweaked the x-axis limits to make sure all the 
     text fit on the plot.)
 
-*   If `check_overlap = TRUE`, overlapping labels will be automatically 
-    removed. The algorithm is simple: labels are plotted in the order they 
-    appear in the data frame; if a label would overlap with an existing point, 
-    it's omitted. This is not incredibly useful, but can be handy. 
+*   The third parameter is `check_overlap`. If `check_overlap = TRUE`, overlapping 
+    labels will be automatically removed from the plot. The algorithm is simple: 
+    labels are plotted in the order they appear in the data frame; if a label 
+    would overlap with an existing point, it's omitted. This is not incredibly useful, 
```

Comment:
@paleolimbot realised that this is actually much more useful than I realised because you can order the labels from most important to least important.